pandas/conftest.py

```diff
@@ -157,6 +157,15 @@ def add_imports(doctest_namespace):
     doctest_namespace["pd"] = pd
 
 
+@pytest.fixture(autouse=True)
+def check_file_leaks():
```

Comment:
could this be in pandas\tests\io\conftest.py

Reply:
Good idea